Oop-1 Question2 Matrix/Matrix.java

```diff
@@ -0,0 +1,184 @@
+/**
+ * This class is designed to work on matrices
+ * @author Gaurav Tak
+ *
+ */
+public class Matrix {
+	private int[][] matrixElement;
+	private int noOfRow;
+	private int noOfColumn;
+
+	public int getNoOfRow() {
+		return noOfRow;
+	}
+
+	public int getNoOfColumn() {
+		return noOfColumn;
+	}
+	
+	public Matrix() {
+		//do nothing
+	}
+	
+	public Matrix(int row, int column) {
+		noOfRow = row;
+		noOfColumn = column;
+		matrixElement = new int[noOfRow][noOfColumn];
+	}
+	
+	public Matrix(int[][] input) {
+		noOfRow = input.length;
+		noOfColumn = input[0].length;
+		matrixElement = new int[noOfRow][noOfColumn];
+		setMatrix(input);
+	}
+	
+	//getters and setters
+	
+	public void setMatrix(int[][] elements) {
+		if (noOfRow == elements.length || noOfColumn == elements[0].length)
```

Comment:
and if there is some reason behind it what happen if this statement goes return false